contributors/devel/sig-testing/testing.md

```diff
@@ -80,12 +69,6 @@ added automatically to these:
 make test WHAT=./pkg/kubelet                # run tests for pkg/kubelet
 ```
 
-Expressed strictly with `go test`, the above command is equivalent to the following:
-
-```sh
-go test ./pkg/kubelet
-```
-
```

Comment:
I think the first one of these was okay, because the assumed context was "base directory of project".